packages/angular-17/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "angular-17",
+  "version": "0.0.0",
```

Comment:
make sure the version is the same as the other packages

Reply:
We are relying on a project inside of Angular. So this one can be named whatever and have whatever version. Take a look on angular-17/projects/components, that one is "in sync".